### ISSUE TITLE
Consider switching connectivity structure from edges to faces

### ISSUE THREAD

**Issue by tsalo** [neurostuff/NiMARE#729]
## Summary

The MATLAB ALE implementation I referenced when writing the Python ALE implementation that eventually was integrated into NiMARE used an edge connectivity structure, which is also the default for SPM. I don't know what other meta-analysis tools (e.g., BrainMap, MKDA) use, but NiMARE users often use NiMARE in conjunction with Nilearn, which uses a face connectivity structure. This can be a problem, as clusters defined by NiMARE tools (e.g., cluster-corrected maps, Jackknife/FocusCounter results) may not match the ones defined by Nilearn (e.g., [get_clusters_table](https://nilearn.github.io/stable/modules/generated/nilearn.reporting.get_clusters_table.html#nilearn.reporting.get_clusters_table)).

Basically, the question is whether we want to improve compatibility with Nilearn at the potential cost of reduced similarity to other meta-analysis tools. Again, I don't know how other tools (beyond SPM and Simon Eickhoff's MATLAB ALE implementation) define clusters.

## Next steps

1. Discuss.
2. If we're happy switching from edge connectivity to face connectivity, then we just need to replace all instances of `ndimage.generate_binary_structure(3, 2)` with `ndimage.generate_binary_structure(3, 1)`.
